client/src/App/Components/SelectSymptom/SelectSymptom.js

```diff
@@ -0,0 +1,185 @@
+import React, { useState, Component } from "react";
```

Comment:
This file is confusing and it is very hard to read .
**Action points : 1. split it up to three files according to your components and functions that you are using `StackedColumn100Chart`  ,`SelectSymptom`, `MoodData`
2.Delete all console logs 
3.Avoid setting state inside the render method because setting a state will render everytime it is called and it might cause performance issues you should research that it is good to know 
4.props you are sending to the picker such as `setStartDate` you are not handling you can get rid of them 
5.imports that you are not using such as `Container` and `Link` you can delete them.**